tests/descriptors/topological.py

```diff
@@ -0,0 +1,164 @@
+import pytest
+from rdkit.Chem import MolFromSmiles, Mol, GetDistanceMatrix
+
+from skfp.descriptors import topological as top
+
+
+def diverse_molecules() -> list[tuple[str, Mol]]:
+    return [
+        ("ethane", MolFromSmiles("CC")),
+        ("ethanol", MolFromSmiles("CCO")),
+        ("carbon_dioxide", MolFromSmiles("O=C=O")),
+        ("benzene", MolFromSmiles("C1=CC=CC=C1")),
+        ("acetic_acid", MolFromSmiles("CC(=O)O")),
+        ("pyridine", MolFromSmiles("c1ccncc1")),
+        ("isobutane", MolFromSmiles("C(C)(C)C")),
+        ("pyrimidine", MolFromSmiles("c1cnc2ncnc12")),
+    ]
```

Comment:
You can also calculate the distance matrices here, and use them in tests: first without passing it (descriptor function computing it internally), then passing. Both results should be the same.